pipeline/test_executor.groovy

```diff
@@ -115,15 +120,28 @@ node("rhel-8-medium || ceph-qe-ci"){
             }
             def cli = getCLI()
             retVal = sharedLib.executeTestSuite(cli, cleanupOnSuccess, cleanupOnFailure)
+            dummy = retVal["result"]
+            retVal.put("status", dummy)
+            print(retVal)
```

Comment:
Yes we do need them since https://github.com/red-hat-storage/cephci/blob/master/pipeline/vars/v3.groovy#L80 is expecting v['status'] which is retval['status'] which we did not had in this case

Reply:
Okay... thanks for the clarification. IMO, we can do
```
retVal.put["status", retVal["result"])
```

Reply:
@psathyan Thanks, I made these changes and below is the log, I intentional aborted the test as our are of interest is in publish results stage which worked fine 
logs - https://ceph-downstream-jenkins-csb-storage.apps.ocp-c1.prod.psi.redhat.com/view/RHCS%20QE/job/sample-rhceph-test-executor/1/console
I have forward you the email too. Please review